setup.py

```diff
@@ -10,45 +10,38 @@ def read(*parts):
         return fp.read()
 
 
-PACKAGE = "phileo"
-NAME = "phileo"
-DESCRIPTION = "a liking app"
-AUTHOR = "Pinax Team"
-AUTHOR_EMAIL = "team@pinaxproject.com"
-URL = "http://github.com/pinax/phileo"
-VERSION = "1.3.1"
-
-
 setup(
-    name=NAME,
-    version=VERSION,
-    description=DESCRIPTION,
+    author="Pinax Team",
+    author_email="team@pinaxproject.com",
+    description="a liking app for Django",
+    name="pinax-likes",
     long_description=read("README.rst"),
-    author=AUTHOR,
-    author_email=AUTHOR_EMAIL,
+    version="1.3.1",
+    url="http://pinax-likes.rtfd.org/",
```

Comment:
Let's make the URL the repo. The docs will be moved to live in pinax docs at `pinax/pinax`